### DISCUSSION TITLE
Open an app to a designated desktop

### DISCUSSION THREAD

**Discussion by eirenicon** [labwc/labwc#1304] [Q&A] [answered]
I am seeking the correct context for the following complex action (perhaps someone can help).  I know what I have below is incorrect, what I don't know is the correct form/ format.

    <keybind key="W-g">
		<item label="geany">
			<action name="Execute"><command>geany</command></action>
			<action name="SendToDesktop" to="" wrap="yes" /></action>
		</item>      
    </keybind>

TIA

**Comment by stefonarch** [comment, marked as answer]
This works here with my apps, opened by any mean the will open on their desktop and get focus:
```
<keybind key="W-g">
	<action name="Execute"><command>geany</command></action>
</keybind>
```

```
<windowRules>
   <windowRule identifier="geany">
        <action name="SendToDesktop" to="xyz"/>
   </windowRule>
</windowRules>
```

**Reply by eirenicon** [reply]
Thank you!
